lib/lita/adapters/slack/api.rb

```diff
@@ -47,6 +47,39 @@ def im_list
           call_api("im.list")
         end
 
+        def conversations_list(types: ["public_channel"], page_limit: nil)
+          result = call_api(
+            "conversations.list",
+            {
+              limit: page_limit,
+              types: types.join(",")
+            }
+          )
+
+          next_cursor = result.dig("response_metadata")&.dig("next_cursor")
```

Comment:
Nit: I think you can `#dig` as a single call to the same result?

Reply:
Yup,  did a bit of a refactor here too